src/octopeer-github/content/EventTracker/MousePositionTracker.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by Mathias on 2016-05-27.
+ */
+class MousePositionTracker {
+
+    /** Private static final for the timeout between logs. */
+    private static get MOUSE_POSITION_TRACKER_TIMEOUT() { return 500; }
+
+    /**
+     * Initialize a MousePositionTracker that contains a MousePositionDatabaseAdaptable.
+     * @param db The DatabaseAdaptable for the Tracker.
+     */
+    constructor(private db: MousePositionDatabaseAdaptable) {
+        const self = this;
+        let $window: JQuery;
```

Comment:
I'm not sure about this, since `window` is already a global variable that we do not want to be shadowed. What do you propose as name?